freqtrade/freqai/spice_rack/lightgbm_config.json

```diff
@@ -0,0 +1,37 @@
+{
+
+    "freqai": {
+        "enabled": true,
+        "purge_old_models": true,
+        "train_period_days": 4,
+        "backtest_period_days": 2,
+        "identifier": "spicy-id",
+        "feature_parameters": {
+            "include_timeframes": [
+                "30m",
+                "1h",
+                "4h"
+            ],
+            "include_corr_pairlist": [
+                "BTC/USD",
+                "ETH/USD"
+            ],
+            "label_period_candles": 20,
+            "include_shifted_candles": 2,
+            "DI_threshold": 0.9,
+            "weight_factor": 0.9,
+            "principal_component_analysis": true,
+            "indicator_periods_candles": [
+                10,
+                20
+            ]
+        },
+        "data_split_parameters": {
+            "test_size": 0,
+            "random_state": 1
+        },
+        "model_training_parameters": {
+            "n_estimators": 800
+        }
```

Comment:
I think you are confused about the target user of the `spice_rack`. `spice_rack` users don't decide the model. The name of the file is `lightgbm_config.json` afterall. They get a pre-made freqai instance under the hood. That is it. 

I don't see why you are trying to give this flexibility - if users want to start setting model params and model types then they can do that by just using normal `FreqAI`. `spice_rack` has one goal **simplicity**. It is for users who don't want to mess with model parameters and configuration files.

Reply:
And by the way, `n_estimators` *does* work for catboost, xgboost, and lightgbm. This config would technically work for all our current regressors/classifiers.

But that doesnt change the fact that we decide the model type too for the user - so it is a moot point.

Reply:
it's not 100% relevant for spice_rack (although i think users may want to pick "run on GPU" if one is available) 
HOWEVER - it is very relevant for regular freqAI :)

Reply:
I don't disagree with the idea of adding flexibility in a single config to switch between models (for normal FreqAI) or for adding GPU support for `spice_rack`. But I don't believe either of these are in the scope of the present PR.

As lightgbm is not compiled by default with GPU - and `spice_rack` is designed to run on lightgbm (since it is the most flexible one of the three, running on cpus, arm, and considered to be the "fastest"), I would leave any integration of further models/GPU spice_rack support to future PRs. 

Goal here is to provide a very simplified interface to an additional indicator. We reiterate in the doc that `spice_rack` should *NOT* be used alone. It is but another indicator to support a strat - nothing more - nothing less. Hence highly efficient and low levels of user attention.